src/libmongoc/doc/mongoc_client_session_get_dirty.rst

```diff
@@ -0,0 +1,33 @@
+:man_page: mongoc_client_session_get_dirty
+
+mongoc_client_session_get_dirty()
```

Comment:
The name is admittedly odd. But the rationale is:
1. `_get` is consistent with other client session methods like `mongoc_client_session_get_server_id`.
2. `dirty` matches terminology from the specification.